cmd/kubeadm/app/phases/upgrade/postupgrade.go

```diff
@@ -34,7 +36,7 @@ import (
 	"k8s.io/klog/v2"
 
 	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
-	kubeadmapiv1 "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta3"
+	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
 	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
```

Comment:
```suggestion
	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
```

we already import the constants package.